dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/S3Utils.java

```diff
@@ -175,21 +175,33 @@ public String getFileName(ResourceType resourceType, String tenantCode, String f
     }
 
     @Override
-    public void download(String tenantCode, String srcFilePath, String dstFile, boolean deleteSource, boolean overwrite) throws IOException {
+    public void download(String tenantCode, String srcFilePath, String dstFilePath, boolean deleteSource, boolean overwrite) throws IOException {
+        File dstFile = new File(dstFilePath);
+        if (dstFile.exists()) {
+            if (overwrite) {
+                if (dstFile.isDirectory()) {
+                    org.apache.commons.io.FileUtils.deleteDirectory(dstFile);
```

Comment:
Please use FileUtils.deleteDirectory instead of this.

Reply:
> 

If so, I would import a conflicting class with org.apache.dolphinscheduler.common.utils.FileUtils

Reply:
We can use `Files.delete` instead of this.

Reply:
> We can use `Files.delete` instead of this.

ok，it has already been updated